admin_manual/release_notes.rst

```diff
@@ -0,0 +1,266 @@
+================================
+ownCloud |version| Release Notes
+================================
+
+Recommended Setup for Running ownCloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ownCloud runs on multiple operating systems and HTTP servers. For best 
+performance, stability, support, and full functionality we recommend:
+
+* Red Hat Enterprise Linux 7
+* MySQL/MariaDB
+* PHP 5.4 +
+* Apache 2.4
+
+Supported Platforms
+^^^^^^^^^^^^^^^^^^^
+
+* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
+* Server: Windows 2008R2; 2012, Linux (Debian 6, RHEL / Centos 6 and 7, Ubuntu 
+  12 and 14, SLES 11, Univention Corporate Server 3.x)
+* Databases: MySQL 5.x; Microsoft SQL Server 2008R2; SQL Server 2012 R2; Oracle 
+  11g
+* Desktop: Windows XP, Windows 7, Mac OS X 10.7+ (64-bit only), Linux (CentOS 
+  6, Ubuntu 10.04 and 11.04+, Fedora 16+, openSUSE 11.4+)
```

Comment:
11.04 is EOL, current Ubuntu versions are the LTS ones 10.04, 12.04 and 14.04 (with 10.04 having EOL in 1 month or so) and 14.10 (support until july 2015) https://wiki.ubuntu.com/Releases